doc/tools/build.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python
+# coding: utf-8
+
+# Builds GD Script API
+# Requires Python 3.4+
+# Run with --help or see print_usage 
+
+import sys
+if(sys.version_info<(3,4)):
+    print("Please upgrade python to version 3.4 or higher.")
+    print("Your version: %s\n" %sys.version_info)
+    sys.exit(1)
+
+from pathlib import Path
+
+# Configure these variables for your system or specify on the command line
+
+GD=str(Path.home()) + "/bin/godot-dev"    # Path to Godot executable
```

Comment:
By default this should better be `../../bin/godot.<platform>.<opt?>.<tools?>.<bits>.exe`, which is where Godot is built. What do you think?

Reply:
Good idea for the path. If the user wants to call Godot, they don't need the first script. They can just use the xmlproc.py script themselves. It's designed for command line or import.